model/sifrovatko.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use \Defuse\Crypto\Key,
+  \Defuse\Crypto\Crypto,
+  \Defuse\Crypto\Exception\CryptoException;
+
+class Sifrovatko
```

Comment:
V této třídě je potřeba opravit code style. Nad rámec věcí co jsou někde napsané ještě prosím nedávat return typehinty a @throws anotace. Na nich sice není nic špatného, ale má prioritu konzistence kódu napříč repem.